editor/package.json

```diff
@@ -72,6 +72,7 @@
 		"@radix-ui/react-tooltip": "^1.0.7",
 		"@radix-ui/react-radio-group": "^1.1.3",
 		"@xterm/addon-fit": "^0.10.0",
+		"@xterm/addon-webgl": "^0.19.0",
```

Comment:
I found only this version available for addon-webgl: "@xterm/addon-webgl": "0.18.0"